var/spack/repos/builtin/packages/libxcrypt/package.py

```diff
@@ -31,6 +31,7 @@ def url_for_version(self, version):
     variant("obsolete_api", default=False, when="@4.4.30:")
 
     patch("truncating-conversion.patch", when="@4.4.30")
+    depends_on("perl-file-open", type=("build", "run"), when="@4.3.31:")
```

Comment:
```suggestion
    depends_on("perl", type="build", when="@4.3.30:")
```

`open.pm` is part of perl itself

Reply:
Thanks.. Made the changes.